### PR TITLE
build(deps): bump versions

### PR DESCRIPTION
**Pull request by driskull**
build(deps): bump versions

### CODE REVIEW

File: src/tests/setupTests.ts [hunk 1]
```diff
@@ -4,3 +4,7 @@ console.error = function (message: any, ...args: any[]) {
   error.call(console, message, ...args); // keep default behaviour
   throw message instanceof Error ? message : new Error(message);
 };
+
+console.warn = () => {
+  // ignore console warnings
```
Comment (jcfranco) [inline src/tests/setupTests.ts:9]: Not sure if warnings should be ignored.

Comment (driskull) [reply src/tests/setupTests.ts:9]: I was trying to get rid of the warning in the `calcite-color` test that uses an invalid value for setting a color. What do you think?

Comment (jcfranco) [reply src/tests/setupTests.ts:9]: I could update the test to intercept the warning message since it is expected.

Comment (driskull) [reply src/tests/setupTests.ts]: Ok. i'll remove the code for getting rid of all warnings :)

Comment (jcfranco) [reply src/tests/setupTests.ts]: OK, I'll update the test.

**Pull request merged**